### ISSUE TITLE
Learner view: independent practice quizzes can be found from a completion modal

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#8538]
### User stories for this issue:

> As a learner, I want to be able to access an independent practice quiz from the completion modal.

|Frame|Explanation|
|--|--|
|![Screen Shot 2021-10-25 at 10 32 36 PM](https://user-images.githubusercontent.com/13563002/138814795-67c8484c-86b5-4b0a-ba24-e31305c3a423.png)|When a learner sees the modal, they should be able to go from the modal to the practice quiz|
|![Screen Shot 2021-10-25 at 10 32 52 PM](https://user-images.githubusercontent.com/13563002/138814821-22d1743e-66c3-4863-91a6-f761a4472e4d.png)|The practice quiz should be rendered when "Move on" is clicked|

### Notes and changes to current designs:
- [Figma for completion modal](https://www.figma.com/file/TfaSTT9gs6fPKhGdNUjffU/Channel-quizzes?node-id=1315%3A12505)
- Routes need to be created
- All strings required should already be in `commonCoreStrings`, `commonCoachStrings`, or `commonLearnStrings`

### Criteria for success:
**Please copy these into your PR for "Steps for Testing"!**

This issue is done when the checkboxes and Gherkin stories below work - Gherkin stories for this need to be created. This will be completed in a separate issue, here: #8539. Please do not open the PR until the Gherkin is updated. The update will be added to this issue when ready.

**Comment by bjester** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Kolibri-Ecosystem-6047e525b8b9d900182e2d6f/issues/learningequality/kolibri/8538?planning-poker) with ZenHub @jredrejo @marcellamaki @sairina @rtibbles @MisRob @nucleogenesis @jamalex

**Comment by rtibbles** [comment]
Fixed in #8645
